src/vs/workbench/contrib/terminal/browser/xterm/commandTrackerAddon.ts

```diff
@@ -332,3 +362,11 @@ export class CommandTrackerAddon extends Disposable implements ICommandTracker,
 		return this._getCommandMarkers().length;
 	}
 }
+
+registerThemingParticipant((theme: IColorTheme, collector: ICssStyleCollector) => {
+	const focusBorderColor = theme.getColor(focusBorder);
+
+	if (focusBorderColor) {
```

Comment:
as some people mentioned in the UX sync, perhaps this should intentionally be different from the focus border so people aren't expecting the focus behavior?

Reply:
we could instead go with the gray that's used for the `go to implementations` decoration

Reply:
This got me to realize it only showed focus border because of my theme:

```json
      "editor.symbolHighlightBackground": "#0000",
      "editor.symbolHighlightBorder": "#399ef4",
```

Here's what it looks like on Dark+ without that:

![image](https://user-images.githubusercontent.com/2193314/158665447-a793d881-7840-4863-9b2e-b03e47637509.png)

Will try that and maybe add theme keys for it too

Reply:
Hmm, but an issue there is the background has to be drawn on top for us, so it would obscure the text...

Reply:
that looks good to me. couldn't you set the transparency?